agent_deploy/kubernetes/sysdig-node-analyzer-daemonset.yaml

```diff
@@ -0,0 +1,252 @@
+# apiVersion: extensions/v1beta1  # If you are in Kubernetes version 1.8 or less please use this line instead of the following one
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sysdig-node-analyzer
+  labels:
+    app: sysdig-node-analyzer
+spec:
+  selector:
+    matchLabels:
+      app: sysdig-node-analyzer
+  updateStrategy:
+    type: RollingUpdate
```

Comment:
What's the advantage of having a RollingUpdate strategy for these components?

Reply:
For daemonsets, RollingUpdate is the only updateStrategy that will update all the pods automatically when the ds changes: https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/#daemonset-update-strategy

 I was also keeping it consistent with the image-analyzer daemonset.